woocommerce/woocommerce-ios
# Fixed issue where no dates not available for selection

Closes: #6089

### Description
Fixed issue where no dates not available for selection In the custom range selection for the orders filters.
If you select the custom start/end dates in a particular order and then try to select again the start or end date it will have all the dates not available for selection.
This was caused because both maximum and a minimum date is set the the datepicker of the DatePickerTableViewCell because it is reused for the start and end date.

### Testing instructions
Steps to reproduce the behavior:

Go to Orders
1. Tap on 'Filters'
2. Tap on 'Date Range'
3. Tap on 'Custom Range'
4. Tap on 'Start Range'
5. Tap on 'End Range'
6. Tap on 'Start Range' again
 
All the dates from the current selection until the end  should be available for selection


Before & after the fix:


https://user-images.githubusercontent.com/96764631/152693843-cff21608-a8e3-4ad9-9b50-e8086edd47a9.mp4


https://user-images.githubusercontent.com/96764631/152693852-ee25e715-2271-4bb5-b591-9ed3c9a0d7ec.mp4


---
